### DISCUSSION TITLE
Entity的小眼睛按钮状态无法及时更新

### DISCUSSION THREAD

**Discussion by terrykingcha** [galacean/discussion#78] [Editor] [answered]
点击小眼睛（二级的有问题，一级的没问题），预览窗口里的Entity已经隐藏了，但是这个小眼睛还是睁开的状态

![May-29-2023 17-56-54](https://github.com/galacean/discussion/assets/1314662/62d4c86f-8a27-43c6-b439-f3d7962eddbd)

**Comment by MrKou47** [comment, marked as answer]
已修复
